src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -1258,7 +1258,7 @@ export default withOnyx<MoneyRequestConfirmationListProps, MoneyRequestConfirmat
     },
     mileageRates: {
         key: ({policyID}) => `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
-        selector: DistanceRequestUtils.getMileageRates,
+        selector: (policy: OnyxEntry<OnyxTypes.Policy>) => DistanceRequestUtils.getMileageRates(policy),
```

Comment:
NAB: I'm mostly curious, why do we need to declare the type of the policy param explicitly?